README.md

```diff
@@ -437,7 +469,7 @@ You should also find a `quote.dat` file generated.
 4. Finally, attest with the Intel Trust Authority service.
 
 ```bash
-trustauthority-cli token -c config.json
+sudo trustauthority-cli token -c config.json
```

Comment:
here also, wonder how i can miss the need of su rights

Reply:
I removed all the `sudo`'s from the guest commands.